ansible/ha-remediation.yml

```diff
@@ -0,0 +1,220 @@
+# This playbook deploys a simple stand alone checker for SAP/HANS.
+
+- hosts: all
+
+  tasks:
+    - name: include load_facts
+      import_role:
+        name: load_facts
+      tags:
+        - always
+
+    - name: include 1.1.1 remediate
+      import_role:
+        name: '1.1.1'
+        tasks_from: main
```

Comment:
Do we need this `main` thing? Is it not redundant?

Reply:
Done.